research/picai/prepare_annotations.py

```diff
@@ -0,0 +1,47 @@
+import argparse
+import os
+
+import SimpleITK as sitk
+
+
+def prepare_annotations(human_annotations_dir: str, ai_annotations_dir: str, annotations_write_dir: str) -> None:
```

Comment:
Maybe a comment that we're writing these to separate annotations sources to a central location etc.